README.md

```diff
@@ -115,7 +115,32 @@ In order to execute CDS logic, the FHIR Clinical Guidelines Implementation guide
 
 ### Run CQL Tests
 
-To run the 1000+ tests to verify functionality of the CCSM CDS, type `npm run test-cql` into a command prompt from within the same directory as this README file. Note that this will require a UMLS account and VSAC API key; please see documentation for the [CQL Testing Framework](https://github.com/AHRQ-CDS/CQL-Testing-Framework) for further instructions. The results from running the tests against the current CQL definitions can be found in [test-results.txt](./test-results.txt).
+To run the 1000+ tests to verify functionality of the CCSM CDS, type `npm run test-cql` into a command prompt from within the same directory as this README file. Note that this will require a UMLS account and VSAC API key; please see the section below, [Setting Up VSAC](#setting-up-vsac), for further instructions. The results from running the tests against the current CQL definitions can be found in [test-results.txt](./test-results.txt).
```

Comment:
Should we add a line at the end of this section (or the next one), saying something like "see [CQL Testing Framework README](https://github.com/AHRQ-CDS/CQL-Testing-Framework?tab=readme-ov-file#cql-testing-framework) for more information"?